stable/xray/templates/xray-server-svc.yaml

```diff
@@ -19,6 +19,11 @@ spec:
 {{- if .Values.server.loadBalancerSourceRanges }}
   loadBalancerSourceRanges:
 {{ toYaml .Values.server.loadBalancerSourceRanges | indent 4 }}
+{{- end }}
+{{- if eq .Values.server.service.type "LoadBalancer" }}
```

Comment:
@lexual I think it would be better to use `and` rather than a nested `if`.
I mean, something like:
```
{{- if and (eq .Values.server.service.type "LoadBalancer") .Values.server.service.loadBalancerIP  }}
loadBalancerIP: {{ .Values.server.service.loadBalancerIP }}
{{- end }}
```

Reply:
have pushed changed that adopts your suggested change here.